open-metadata-implementation/access-services/asset-catalog/asset-catalog-server/src/main/java/org/odpi/openmetadata/accessservice/assetcatalog/exception/AssetCatalogException.java

```diff
@@ -76,75 +105,4 @@ public AssetCatalogException(int httpCode,
         this.reportedUserAction = userAction;
         this.reportedCaughtException = caughtError;
     }
-
-
-    /**
-     * Return the HTTP response code to use with this exception.
-     *
-     * @return reportedHTTPCode
-     */
-    public int getReportedHTTPCode() {
-        return reportedHTTPCode;
-    }
-
-    /**
-     * The class that created this exception.
-     *
-     * @return reportingClassName
-     */
-    public String getReportingClassName() {
-        return reportingClassName;
-    }
-
-
-    /**
-     * The type of request that the class was performing when the condition occurred that resulted in this
-     * exception.
-     *
-     * @return reportingActionDescription
-     */
-    public String getReportingActionDescription() {
-        return reportingActionDescription;
```

Comment:
This will need to be turned back into an explicit method when we add localization.